app/views/shared/_navbar.html.erb

```diff
@@ -48,7 +48,7 @@
   <div class="navbar-wagon-second hidden-xs hidden-sm font-oswald">
     <div>
       En ce moment :
-      <% topics.each do |topic| %>
```

Comment:
instead of changing this partial, you should change `<%= render 'shared/navbar', topics: @topics_nav %>` with `<%= render 'shared/navbar', topics: @topics %>` in [application.html.erb](https://github.com/guillaumecabanel/hedgenews/blob/master/app/views/layouts/application.html.erb)